com.unity.render-pipelines.high-definition/Editor/Lighting/HDLightExplorerExtension.cs

```diff
@@ -525,6 +540,107 @@ protected virtual LightingExplorerTableColumn[] GetPlanarReflectionColumns()
             };
         }
 
+        protected virtual LightingExplorerTableColumn[] GetProbeVolumeColumns()
+        {
+            return new[]
+            {
+                new LightingExplorerTableColumn(LightingExplorerTableColumn.DataType.Name, HDStyles.Name, null, 200),                                       // 0: Name
+                new LightingExplorerTableColumn(LightingExplorerTableColumn.DataType.Custom, HDStyles.DrawProbes, "parameters", 35, (r, prop, dep) =>       // 1: Draw Probes
+                {
+                    SerializedProperty drawProbes = prop.FindPropertyRelative("drawProbes");
+                    EditorGUI.PropertyField(r, drawProbes, GUIContent.none);
+                }, (lhs, rhs) =>
+                {
+                    return lhs.FindPropertyRelative("drawProbes").boolValue.CompareTo(rhs.FindPropertyRelative("drawProbes").boolValue);
+                }),
+                new LightingExplorerTableColumn(LightingExplorerTableColumn.DataType.Custom, HDStyles.DebugColor, "parameters", 75, (r, prop, dep) =>       // 2: Debug Color
+                {
+                    SerializedProperty debugColor = prop.FindPropertyRelative("debugColor");
+                    EditorGUI.PropertyField(r, debugColor, GUIContent.none);
+                }, (lhs, rhs) =>
+                {
+                    float lh, ls, lv, rh, rs, rv;
+                    Color.RGBToHSV(lhs.FindPropertyRelative("debugColor").colorValue, out lh, out ls, out lv);
+                    Color.RGBToHSV(rhs.FindPropertyRelative("debugColor").colorValue, out rh, out rs, out rv);
+                    return lh.CompareTo(rh);
```

Comment:
Just out of curiosity, what's this luminance comparison for? Guessing it's for sorting by column - colors are sorted by luminance?

Reply:
Yes, we sort by the Hue. This is how we sort all colors in the explorer, but since the value is kind of "hidden", it has to be reimplemented. I didn't keep this in mind when I did the original version, and is trying to figure out a better way to refactor it :)